release/cli/pkg/constants/constants.go

```diff
@@ -62,13 +62,4 @@ const (
 
 	// KMS key alias
 	KmsKey = "arn:aws:kms:us-west-2:857151390494:alias/signingEKSABundlesKey"
-
```

Comment:
I have moved these constants to eks-anywhere constants package since these are used for both signing and verification purpose and to keep a single source of truth.